File: 46- ত্রিভুজের ক্ষেত্রফল.c
#include<stdio.h>
int main()
{

    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c;
        scanf("%d%d%d",&a,&b,&c);
        double s = (a+b+c)/2.0;
        double sum = sqrt(s*(s-a)*(s-b)*(s-c));
        printf("Area = %0.3lf\n",sum);

    }
    return 0;


}

File: 46- ত্রিভুজের ক্ষেত্রফল.py
t = int(input())

import math

while t!=0:
    a,b,c = input().split()
    a = int(a)
    b = int(b)
    c = int(c)

    s = (a+b+c)/2

    su = math.sqrt(s*(s-a)*(s-b)*(s-c))

    print("Area =","{:.3f}".format(su))

    t-=1
